agent/containermetadata/types.go

```diff
@@ -130,34 +132,38 @@ type Metadata struct {
 }
 
 // metadataSerializer is an intermediate struct that converts the information
-// in Metadata into information to encode into JSOn
+// in Metadata into information to encode into JSON
 type metadataSerializer struct {
-	Cluster              string            `json:"Cluster,omitempty"`
-	ContainerInstanceARN string            `json:"ContainerInstanceARN,omitempty"`
-	TaskARN              string            `json:"TaskARN,omitempty"`
-	ContainerID          string            `json:"ContainerID,omitempty"`
-	ContainerName        string            `json:"ContainerName,omitempty"`
-	DockerContainerName  string            `json:"DockerContainerName,omitempty"`
-	ImageID              string            `json:"ImageID,omitempty"`
-	ImageName            string            `json:"ImageName,omitempty"`
-	Ports                []api.PortBinding `json:"PortMappings,omitempty"`
-	Networks             []Network         `json:"Networks,omitempty"`
-	MetadataFileStatus   MetadataStatus    `json:"MetadataFileStatus,omitempty"`
+	Cluster                string            `json:"Cluster,omitempty"`
+	ContainerInstanceARN   string            `json:"ContainerInstanceARN,omitempty"`
+	TaskARN                string            `json:"TaskARN,omitempty"`
+	TaskDefinitionFamily   string            `json:"TaskDefinitionFamily,omitempty"`
+	TaskDefinitionRevision int             `json:"TaskDefinitionRevision,omitempty"`
```

Comment:
I think this has to be `*int`. `0` is still a valid int and `omitempty` would not apply to it.